apps/calendar/js/backend/calendar_service.js

```diff
@@ -1,88 +1,104 @@
+define(function(require, exports, module) {
 'use strict';
-importScripts('../ext/alameda.js');
 
-require.config({
-  baseUrl: '/js/backend',
-  paths: {
-    ext: '/js/ext'
-  }
-});
-
-require(['ext/threads'], threads => {
-  var service = threads.service('calendar');
-
-  service.method('start', () => {
-    // TODO
-    console.log('start');
+var Db = require('db');
+var co = require('ext/co');
+var threads = require('ext/threads');
+
+var db = new Db('b2g-calendar');
+var loadDb = db.load();
```

Comment:
should we wait until some method is called before triggering the `db.load`? to me it feels weird to trigger an action automatically on the "module body"